src/store/printMyBarcode/actions.js

```diff
@@ -1,74 +1,56 @@
 import { handleResponse } from '@/api/ResponseHelper'
 
-const printJob = async ({ getters }, params) => {
-  const request = getters.printJobRequest
-
-  const labelTemplateName = getters.tubeLabelTemplateName
-
-  const payload = createPrintJobJson(params, labelTemplateName)
+/*
+  TODO. The only action we have is createPrintJob
+  Question: Do we need this in the store at all?
+  The action does not relate to the store so is maybe better as a service.
+*/
```

Comment:
Yep, would agree with this comment. Doesn't need to be in the store as it isn't handling common state which needs to be accessible throughout the application